### PR TITLE
Document `is_parameter` in the manual (besides api ref)

### PR DESCRIPTION
**Pull request by joaquimg**

### CODE REVIEW

**Comment by joaquimg** [comment]
doc errors do not seem related, or I missed something in the log

File: docs/src/manual/variables.md [hunk 1]
```diff
@@ -1355,6 +1355,20 @@ julia> parameter_value.(p)
...
+
+```jldoctest nonlinear_parameters
+julia> is_parameter(p[1])
+true
+
+julia> is_parameter(x)
+false
+
+julia> ParameterRef(p[2])
+p[2] ∈ MathOptInterface.Parameter{Float64}(3.0)
```
Comment (odow) [inline docs/src/manual/variables.md:1369]: It's probably the `∈`. I'll take a look. Debugging the latex can be tricky.

Comment (joaquimg) [reply docs/src/manual/variables.md:1369]: Weird, I used the same as the one from line 1275

**Comment by odow** [comment]
Hmm the logs have
```
Unable to find image 'juliadocs/documenter-latex:0.1' locally
docker: Error response from daemon: Head "https://registry-1.docker.io/v2/juliadocs/documenter-latex/manifests/0.1": unauthorized: incorrect username or password.
See 'docker run --help'.
Error response from daemon: No such container: latex-container
```

**Pull request merged**